urllib3/util/__init__.py

```diff
@@ -24,6 +24,10 @@
     split_first,
     Url,
 )
+from .selectors import (
+    wait_for_read,
+    wait_for_write
+)
```

Comment:
At this point I think we shouldn't bother doing this cross-import thing. Either we want our backport to maintain the same interface as the stdlib, in which case the implementation of these two functions should be moved fully into `util`, or we don't care, in which case our code should just import them directly from `selectors`. Up to you @SethMichaelLarson, but we should aim to do one of them.